help/de/Help_ch_GUIMenus.xml

```diff
@@ -91,14 +87,33 @@
           <listitem>
             <para>Enthält Schaltflächen (Knöpfe) für den Zugriff auf typische kontextbezogene Funktionen.
             </para>
+
+            <tip>
+              <para>Die Darstellung der Werkzeugleiste (Größe der Symbole, Text unter dem Symbol uvm.) kann nicht
+                direkt in &app; an Ihre Anforderungen angepasst werden. Wenn Sie das Aussehen von
```

Comment:
Vielleicht etwas positiver ausdrücken?

Das gesamte Erscheinungsbild der [wp-link] CSS-basierte GnuCash-Oberfläche läßt sich über die Konfiguration von [wp-link] GTK gestalten. …

Reply:
Stimmt, ist doof so!

Wie allgemeingültig ist der Hinweis auf CSS und GTK3 eigentlich? Habe ich das richtig verstanden, dass die M$-Nutzer hiervon nicht profitieren können?
Langfristig müsste das ja in das Kapitel "Customize"

Reply:
> Wie allgemeingültig ist der Hinweis auf CSS und GTK3 eigentlich? Habe ich das richtig verstanden, dass die M$-Nutzer hiervon nicht profitieren können?

"Echte" M$-Nutzer vielleicht nicht, da das offizielle Programm zum Zusammenklicken vieler GTk-Einstellungen (noch) nicht von GTK2 auf 3 portiert wurde. Wer aber schon mal an einer Website gebastelt hat oder in Word Dokumentvorlagen nutzt, sollte eigentlich keine Probleme haben, ein paar .ini- und .css-Dateien zu editieren.

> Langfristig müsste das ja in das Kapitel "Customize"

Die Details ja, aber wegen der viel zu großen Abstände zwischen den Ikonen fällt mir das immer hier ein.